R/geom-function.R

```diff
@@ -18,14 +18,20 @@
 #'   geom_function(fun = dnorm, colour = "red")
 #'
 #' # To plot functions without data, specify range of x-axis
-#' base <- ggplot() + xlim(-5, 5)
+#' base <-
+#'   ggplot() +
+#'   xlim(-5, 5)
```

Comment:
One question is a like break after `<-` should be added or not, but I think either is fine.

c.f. https://style.tidyverse.org/pipes.html#assignment-2